docs/OV_Runtime_UG/supported_plugins/ARM_CPU.md

```diff
@@ -0,0 +1,76 @@
+# Arm&reg; CPU device {#openvino_docs_OV_UG_supported_plugins_ARM_CPU}
+
+
+## Introducing the Arm&reg; CPU Plugin
+The ARM&reg; CPU plugin is developed in order to enable deep neural networks inference on Arm&reg; CPU, using [Compute Library](https://github.com/ARM-software/ComputeLibrary) as a backend.
+
+The Arm&reg; CPU plugin is not a part of the Intel® Distribution of OpenVINO™ toolkit and is not distributed in pre-built form. To use the plugin, it should be buid from source code. Plugin build procedure is described on [How to build Arm&reg; CPU plugin](https://github.com/openvinotoolkit/openvino_contrib/wiki/How-to-build-ARM-CPU-plugin). 
+
+The set of supported layers is defined on [Operation set specification](https://github.com/openvinotoolkit/openvino_contrib/wiki/ARM-plugin-operation-set-specification).
+
+
+## Supported inference data types
+The Arm&reg; CPU plugin supports the following data types as inference precision of internal primitives:
+
+- Floating-point data types:
+  - f32
+  - f16
+- Quantized data types:
+  - i8
+
+
+> **NOTE**: i8 support is experimental.
+
+[Hello Query Device C++ Sample](../../../samples/cpp/hello_query_device/README.md) can be used to print out supported data types for all detected devices.
+
+
+## Supported properties
+The plugin supports the properties listed below.
+
+### Read-write properties
+All parameters must be set before calling `ov::Core::compile_model()` in order to take effect or passed as additional argument to `ov::Core::compile_model()`
+
+- ov::enable_profiling
```

Comment:
I suppose you also support streams, pinning, etc
@apankratovantonp  please, provide a full list of properties.

Reply:
Thus these options are in the supported list they do not provide expected threading control. We even test only in latency (sync) mode only. I am going to enable tbb threading backend and provide full threading control as in the CPU plugin. It won't give any valuable throughput gain but give full threading control. So we can claim support of these options.